flybywiresim/aircraft
# fix(hoppie): Updated Hoppie Poll rate

Fixes #[issue_no]

## Summary of Changes

Changes the polling rate to Hoppie Network from a low amount to the recommended 45-75 seconds (randomized) to comply with rate limits. 

## Additional context

Rate limits requests can be found here: https://www.hoppie.nl/acars/system/tech.html
& here: https://www.hoppie.nl/acars/system/cpdlc.html
Quoting: 
> Many clients have set their poll rate at 5 seconds and this is simply never needed, please SET IT TO A MINUTE.

> It is heavily recommended to poll once between every 45 and 75 seconds, randomly timed so that the average server load is constant. If you expect a message, such as right after you sent a request, you may increase the polling rate to once per 20 seconds for a short time.


Discord username (if different from GitHub): alepouna

## Testing instructions

1. Spawn in a runway/gate 
2. Make sure you have Hoppie integration enabled (https://docs.flybywiresim.com/fbw-a32nx/feature-guides/hoppie/) and available (align ADIRS, weather to VATSIM, etc. as docs detail)
3. Create a flight number in INIT A 
4. Test you can receive Hoppie requests by sending a TELEX via MDCU AOC MENU to your own callsign. 
5. You should receive this message within 45-75 seconds after sending it. 
6. Keep sending requests ever so often. You should NOT receive a COMPANY MESSAGE by SYSTEM warning you about the rate limit, if everything is correct. You should test this for at least 5 minutes. 


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
